packages/lib-user/src/components/shared/MainContent/MainContent.js

```diff
@@ -291,7 +292,7 @@ function MainContent({
             margin={{ top: 'small' }}
           >
             <StyledCertificateButton
-              forwardedAs='a'
+              forwardedAs={Link}
```

Comment:
Noting here that `forwardedAs='a'` does nothing on a Grommet Button. Grommet automatically renders Button as a link, if it has a `href` attribute.